src/oop2.py

```diff
@@ -30,3 +39,6 @@ def __init__(self, num_wheels):
 # Go through the vehicles list and call drive() on each.
 
 # TODO
+
+for vehicle in vehicles:
+    print(vehicle.drive())
```

Comment:
I don't think you need this "print" since the drive method already prints. I think vehicle.drive() would work.